install_cexts.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+import os
+import subprocess
+import argparse
+import imp
+
+
+DIST_CEXT = 'kolibri/dist/cext'
+PYPI_DOWNLOAD = 'https://pypi.python.org/simple/{package_name}'
+
+
+"""
+Check if requests and bs4 modules are installed.
+"""
+def _install_requests_and_bs4_if_needed():
+    try:
+        imp.find_module('requests')
+        print ('requests module exists')
+    except:
+        subprocess.call(['pip', 'install', 'requests'])
+
+    try:
+        imp.find_module('bs4')
+        print ('bs4 module exists')
+    except:
+        subprocess.call(['pip', 'install', 'bs4'])
+
+
+"""
+Distribute packages into folders according to the platform.
+"""
+def get_path_with_arch(platform, path):
```

Comment:
If you move the docstrings into the function body, Sphinx will pick it up. Like so:

```
def get_path_with_arch(platform, path):
    """
    Distribute packages into folders according to the platform.
    """
```